docs/modules/concepts/pages/configuration.adoc

```diff
@@ -0,0 +1,34 @@
+= Configuration
+
+== Overview
+
+An important aspect to understand about James is the idea of "configuration".
+Sometimes "configuration" means different things to different people, so we
+wanted to ensure that what we mean by the concept is clear and precise, 
+because in order operate a James server, you will necessarily need to 
+configure it.
+
+A "*Configuration*" is a https://martinfowler.com/bliki/ValueObject.html[value object]
+that communicates to a James server how the xref:user/index.adoc#_operator[Operator] wishes the
+server to behave. It is very similar to an interface, but yet it is not quite an interface
```

Comment:
I don't find that at all similar to an interface or I don't understand what you mean by "interface"

Reply:
Ok, I'll give it a shot.

An "interface" is essentially, conceptually speaking, some "surface" between which the inside of a system communicates with the outside. Except for an interface, normally it should not be possible to communicate with the system. Maybe this dates me a bit, but take the example of a DVD player, and for this example pretend we lost the remote. The "interface" is the collection of buttons and stuff that you would press. It is in this case literally the surface that connects the inside of the machine from the outside.

A configuration also connects the inside from the outside, but it is used for a different purpose and has a different lifecycle.

Does that make sense?

Reply:
If we step back a bit, are these extra notions really necessary or are they creating more confusion by building more and more context?

Reply:
Which "extra notions" are you referring to?

I don't think you mean the idea of configuration itself, because I don't think you mean either of:
 - We remove all configuration from James 👎🏻
 - We provide a means of interacting with the system that either we don't fully understand or we can't explain 👎🏻

Can you explain a bit more?